nftstorage/nft.storage
# ci: github actions dont run release-please unless push-to-main

Motivation:
* https://github.com/nftstorage/nft.storage/issues/1242
* second time (last attempt [reverted](https://github.com/nftstorage/nft.storage/pull/1697)) 

How to Test:
* once this is merged, it should prevent this other PR from breaking: https://github.com/nftstorage/nft.storage/pull/1723
  * 'proof' that that pr would work if not from fork: https://github.com/nftstorage/nft.storage/pull/1725
